capstone/src/main/webapp/script.js

```diff
@@ -374,6 +375,9 @@ async function loadWordcloud() {
   const list = data.map(function(x) {
     return [x.content, x.weight];
   });
+  const list2 = data.map(function(x) {
+    return [x.content, x.weight/7];
```

Comment:
Why divide the weights by 7?

Reply:
To adjust the font size on the smaller wordcloud